### PR TITLE
Bugfix UnifiedNum math operations

### PR DESCRIPTION
**Pull request by elpiel**

### CODE REVIEW

File: primitives/src/unified_num.rs [hunk 1]
```diff
@@ -78,17 +169,51 @@ impl TryFrom<String> for UnifiedNum {
...
+            div_unified_num_to_ratio(self, other).expect("Failed to create ratio for div_floor");
+
+        let whole_number = ratio
+            .checked_div(&Ratio::from_integer(UnifiedNum::MULTIPLIER))
+            .expect("Should divide with Multiplier for div_floor");
+
+        UnifiedNum::from_whole(whole_number.to_integer())
     }
 
+    /// This method returns the inner [`u64`] representation of the [`UnifiedNum`].
```
Comment (simzzz) [inline primitives/src/unified_num.rs]: The comment is wrong

File: sentry/src/routes/units_for_slot_test.rs [hunk 2]
```diff
@@ -0,0 +1,761 @@
+use crate::{platform::PlatformApi, test_util::setup_dummy_app};
```
Comment (simzzz) [inline sentry/src/routes/units_for_slot_test.rs]: I think this file should belong to another PR

**Review by simzzz — CHANGES_REQUESTED** [review]

File: primitives/src/unified_num.rs (hunk 1 above)
Comment (elpiel) [reply primitives/src/unified_num.rs]: Good catch

File: sentry/src/routes/units_for_slot_test.rs (hunk 2 above)
Comment (elpiel) [reply sentry/src/routes/units_for_slot_test.rs]: Hmm yeah, I don't know how that got here...

**Review by simzzz — APPROVED** [review]

**Pull request merged**